docs/reference/cat/transforms.asciidoc

```diff
@@ -59,33 +59,36 @@ Valid columns are:
 `changes_last_detection_time`, `cldt`:::
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=checkpointing-changes-last-detected-at]
```

Comment:
```suggestion
(Default)
include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=checkpointing-changes-last-detected-at]
```

I think the `(Default)` indicates if the column is there by default. These documented columns and the current defaults should sync up.